game/src/story/vwf.asm

```diff
@@ -0,0 +1,743 @@
+SECTION "VWF Variables", WRAMX[$DC00], BANK[$1]
```

Comment:
Could you move these accordingly:
* the variables should be in game/src/patch/wram.asm for now
* The code itself can be in game/src/patch/ as vwf.asm

'story' (for lack of a better name) is meant to be closer to the original, as it simplifies merges at times.